generator-settings/lineLength.md

```diff
@@ -0,0 +1,21 @@
+If you make an email HTML, there is a limit on the number of characters in a line. It is possible to minify your code into one line, but some email clients may have trouble parsing it and that can cause a lot of issues. For example if any transfer agent or email client recieves HTML code with longer lines that it can't handle, the line will be broken as it moves through the code at a possibly arbitrary point. If that break happens in the middle of an HTML tag, it could make the tag unreadable. From that point various kind of rendering issues can occur, mostly display errors that can affect the whole layout. 
```

Comment:
I think it's more likely that the ESP has a line length limit. I'm not sure about the email clients.